### PR TITLE
update unstaking

### PR DESCRIPTION
**Pull request by Jibz1**
https://user-images.githubusercontent.com/8676844/213318810-019eb893-8d68-415f-b358-10374c1db48d.mov

### CODE REVIEW

File: apps/wallet/src/ui/app/staking/stake/StakeForm.tsx [hunk 1]
```diff
@@ -52,17 +50,26 @@ function StakeForm({
         coinBalance -
         BigInt(coinType === SUI_TYPE_ARG ? DEFAULT_GAS_BUDGET_FOR_STAKE : 0);
 
-    const maxTokenFormated = useFormatCoin(coinBalanceMinusGas, coinType);
+    const maxTokenFormate = useFormatCoin(coinBalanceMinusGas, coinType);
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/stake/StakeForm.tsx:53]: Why changing this to a typo-version of the variable?

File: apps/wallet/src/ui/app/staking/stake/StakeForm.tsx [hunk 2]
```diff
@@ -52,17 +50,26 @@ function StakeForm({
...
 
-    const maxToken = maxTokenFormated[0];
-    const queryResult = maxTokenFormated[2];
+    const maxToken = maxTokenFormate[0];
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/stake/StakeForm.tsx:55]: Didn't notice this but you should destructure these out instead.

File: apps/wallet/src/ui/app/staking/stake/StakeForm.tsx [hunk 3]
```diff
@@ -52,17 +50,26 @@ function StakeForm({
...
+    const queryResult = maxTokenFormate[2];
     const [coinDecimals] = useCoinDecimals(coinType);
+    const tokenBalance = useFormatCoin(coinBalance, coinType)[0];
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/stake/StakeForm.tsx:58]: Should destructure these out.

**Review by Jordan-Mysten — APPROVED** [review]
Some nits we should address at some point but approving to unblock. also the "Enter the amount of sui to unstake" copy looks weird given you can't change the amount.

**Comment by mystie711** [comment]
Could you please update this with a video of latest when you think you are done so i could review it @Jibz1 ?

**Pull request merged**